### DISCUSSION TITLE
Redis healthcheck?

### DISCUSSION THREAD

**Discussion by vttranlina** [linagora/tmail-backend#771] [Q&A]
recently, I realized case that when the Redis server restarted with a new password, the tmail server was still normal without any new update (maybe have no new request to Redis in same that time).
but if we restart tmail, it will start fail, because it cannot init mailets that depend on redis (ex: PerRecipientRateLimit)

what do you think?

**Comment by quantranhong1999** [comment]
Yes I think that is a good idea.

**Comment by chibenwa** [comment]
:+1:

**Comment by vttranlina** [comment]
implemented
